templates/shared/common_install_shell_artifacts.gsl

```diff
@@ -508,14 +508,22 @@ done
 
 .endmacro # define_read_parameters
 .
+.macro define_handle_help_line_option()
+.   heading2("Handle help line option")
+if [[ $DISPLAY_HELP ]]; then
+    display_help
+    exit 0
+fi
+
+.endmacro #define_handle_help_line_option
+.
 .macro define_icu(install)
 .   define my.install = define_icu.install
 .   define my.url = get_icu_url(my.install)?
 .   if (!defined(my.url))
 .       #abort "A version of icu is not defined."
 .       return
 .   endif
-.   heading2("ICU archive.")
```

Comment:
Why are these headings removed?

Reply:
Hi @evoskuil. 

>Why are these headings removed?

As described above:
>The comments bring no benefit. Clarity through the comments is questionable and relative. The current comments duplicate the code. 

I see no value or profit in the comments.

Reply:
Apparently they were included because someone felt otherwise. These type of organizational comment blocks are used in most generated code and throughout more lengthy sources. So it’s also a question of consistency. I’d prefer not to go down the path of making such changes, as people will never agree on style.

Reply:
>as people will never agree on style

This is true. 

If there are no standards / styles then there are the following possibilities in my view:
1. We rely on what already exists. 
2. We do as we like. 
3. We define these standards. 

The second variant leads to chaos. The first option is not always obvious and understandable. The third option seems to be the best, even if it makes some unhappy.

I will put these headers back first.

However, in general I do not think that the standards for the quality of various scripts would be worse or less important than the standards set by "more serious" languages, which we map the business logic. That's the reason why I've dealt with the script at all.

Reply:
>I will put these headers back first.

Done.